src/multicellular_stage/editor/CellBodyPlanEditorComponent.cs

```diff
@@ -1403,6 +1403,17 @@ private void SetSelectionMenuTab(string tab)
 
         GUICommon.Instance.PlayButtonPressSound();
 
+        // Don't change the tab if there's an in-progress action
+        if (CanCancelAction)
+        {
+            ToolTipManager.Instance.ShowPopup(Localization.Translate("TAB_CHANGE_BLOCKED_WHILE_ACTION_IN_PROGRESS"),
+                1.5f);
+
+            ApplySelectionMenuTab();
+
```

Comment:
This is 4 lines of duplication if counting the comment. So I think that this is enough stuff to warrant making a helper method to ensure that someone doesn't edit one of the comments for example adding more context or something like that. Other than that and the merge conflicts I think this PR is ready.